tests/e2e/util.go

```diff
@@ -1914,15 +1958,18 @@ func pvcHealthAnnotationWatcher(ctx context.Context, client clientset.Interface,
 //wait for ESX to be up
 func checkHostStatus(ip string) error {
 	framework.Logf("host ip %v", ip)
+	if ip == "" {
```

Comment:
Instead of returning nil, why don't you fail here if `ip` is empty?

Reply:
Sure, we can fail if `ip` is empty and I have updated the same